Airports/EGKR/SMR/Geo.txt

```diff
@@ -1,63 +1,17 @@
-; - HELI4
-N051.12.47.711 W000.08.45.560 N051.12.47.564 W000.08.45.547 smrWhite
-N051.12.47.564 W000.08.45.547 N051.12.47.570 W000.08.45.497 smrWhite
-N051.12.47.570 W000.08.45.497 N051.12.47.678 W000.08.45.497 smrWhite
-N051.12.47.678 W000.08.45.497 N051.12.47.677 W000.08.45.393 smrWhite
-N051.12.47.677 W000.08.45.393 N051.12.47.717 W000.08.45.383 smrWhite
-N051.12.47.717 W000.08.45.383 N051.12.47.711 W000.08.45.560 smrWhite
-; - HELI5_
-N051.12.47.724 W000.08.45.182 N051.12.47.727 W000.08.45.130 smrWhite
-N051.12.47.727 W000.08.45.130 N051.12.47.630 W000.08.45.104 smrWhite
-N051.12.47.630 W000.08.45.104 N051.12.47.633 W000.08.44.985 smrWhite
-N051.12.47.633 W000.08.44.985 N051.12.47.729 W000.08.44.996 smrWhite
-N051.12.47.729 W000.08.44.996 N051.12.47.727 W000.08.44.934 smrWhite
-N051.12.47.727 W000.08.44.934 N051.12.47.499 W000.08.44.902 smrWhite
-N051.12.47.499 W000.08.44.902 N051.12.47.499 W000.08.44.959 smrWhite
-N051.12.47.499 W000.08.44.959 N051.12.47.590 W000.08.44.976 smrWhite
-N051.12.47.590 W000.08.44.976 N051.12.47.588 W000.08.45.091 smrWhite
-N051.12.47.588 W000.08.45.091 N051.12.47.493 W000.08.45.089 smrWhite
-N051.12.47.493 W000.08.45.089 N051.12.47.495 W000.08.45.149 smrWhite
-N051.12.47.495 W000.08.45.149 N051.12.47.724 W000.08.45.182 smrWhite
-; - HELI6
-N051.12.47.748 W000.08.44.738 N051.12.47.748 W000.08.44.541 smrWhite
-N051.12.47.748 W000.08.44.541 N051.12.47.594 W000.08.44.530 smrWhite
-N051.12.47.594 W000.08.44.530 N051.12.47.590 W000.08.44.597 smrWhite
-N051.12.47.590 W000.08.44.597 N051.12.47.701 W000.08.44.597 smrWhite
-N051.12.47.701 W000.08.44.597 N051.12.47.699 W000.08.44.727 smrWhite
-N051.12.47.699 W000.08.44.727 N051.12.47.748 W000.08.44.738 smrWhite
\ No newline at end of file
+N051.12.54.362 W000.07.57.712 N051.12.54.201 W000.07.58.044 smrYellow ;A1
+N051.12.56.356 W000.08.02.504 N051.12.56.111 W000.08.02.475 smrYellow ;A2
+N051.12.50.861 W000.07.56.462 N051.12.50.662 W000.07.56.082 smrYellow ;B1
+N051.12.46.217 W000.08.00.610 N051.12.46.131 W000.08.00.183 smrYellow ;B2
+N051.12.40.237 W000.08.24.178 N051.12.39.816 W000.08.23.994 smrYellow ;C1
+N051.12.45.801 W000.07.55.779 N051.12.45.605 W000.07.55.604 smrYellow ;C2
+N051.12.38.805 W000.08.29.899 N051.12.39.220 W000.08.30.088 smrYellow ;D1
+N051.12.37.207 W000.08.50.164 N051.12.37.321 W000.08.49.785 smrYellow ;D2
+N051.12.49.414 W000.08.45.405 N051.12.49.372 W000.08.45.051 smrYellow ;D3
+N051.13.07.234 W000.08.23.264 N051.13.07.067 W000.08.23.691 smrYellow ;E1
+N051.12.55.539 W000.08.20.487 N051.12.55.307 W000.08.20.777 smrYellow ;E2
+N051.12.46.608 W000.08.41.071 N051.12.46.705 W000.08.40.555 smrYellow ;G1
+N051.12.50.345 W000.08.34.962 N051.12.50.472 W000.08.34.149 smrYellow ;G2
+N051.13.04.113 W000.08.28.426 N051.13.03.649 W000.08.28.458 smrYellow ;G3
+N051.13.08.202 W000.08.31.145 N051.13.07.876 W000.08.30.814 smrYellow ;H2
+N051.13.09.019 W000.08.28.439 N051.13.08.590 W000.08.28.367 smrYellow ;H1
```

Comment:
H1 before H2 please

Reply:
I am blind ok, I need a computer to check everything for me haha